app/views/manage_assessments/coverages/new.html.erb

```diff
@@ -18,6 +18,7 @@
 
   <%= form.simple_fields_for :outcome_coverages do |fields| %>
     <%= fields.input :outcome_id,
+      selected: set_selected_attribute_with_outcome_id_if_present(@outcome),
```

Comment:
I believe this should happen automatically by virtue of the `outcome_id` already being set on the outcome coverage. What happens if you remove this and the the associated helper?